### PR TITLE
created media route

### PR DESCRIPTION
**Pull request by Eoghan-K**

### CODE REVIEW

File: routes/media.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+			// Get selectedMovie
+			selectedMovie = JSON.parse(response);
+			console.log(JSON.parse(response));
+			return rp(options2)
+		})
+		.then(function (response) {
+			// Get latestMovies
+			relatedMovies = JSON.parse(response).results;
+			res.render('pages/media', {
+				activeLink: "home",
```
Comment (larfoley) [inline routes/media.js]: pass in media type to view.

File: routes/media.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+var router = express.Router();
+var https = require('https');
+var request = require("request");
+var rp = require('request-promise');
+
+var options2 = {
+	method: 'GET',
+	url: 'https://api.themoviedb.org/3/discover/movie',
+	qs: {
+		page: '1',
```
Comment (larfoley) [inline routes/media.js]: change tab to spaces(2)

File: routes/media.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+};
+
+router.get('/', function (req, res, next) {
+  var id = req.query.id;
+  var mediaType = req.query.media_type;
+  var media, relatedMovies;
+
+  rp({
+      method: 'GET',
+      url: 'https://api.themoviedb.org/3/movie/' + id,
```
Comment (larfoley) [inline routes/media.js]: Can you change `/movie/` to be a variable with the movie type

File: routes/media.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+      return rp(options2)
+    })
+    .then(function (response) {
+      // Get latestMovies
+      relatedMovies = JSON.parse(response).results;
+      res.render('pages/media', {
+        activeLink: "home",
+        media: media,
+        mediaType: mediaType,
+        relatedMovies: relatedMovies
```
Comment (larfoley) [inline routes/media.js]: change relatedMovies to related media

File: routes/media.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
+var config = require('../config');
+var express = require('express');
+var router = express.Router();
+var https = require('https');
+var request = require("request");
+var rp = require('request-promise');
+
+var options2 = {
+  method: 'GET',
+  url: 'https://api.themoviedb.org/3/discover/movie',
```
Comment (larfoley) [inline routes/media.js]: change the url to this /movie/{movie_id}/similar

File: routes/media.js (hunk 4 above)
Comment (Eoghan-K) [reply routes/media.js]: on it

**Pull request merged**